catalog-item.html

```diff
@@ -10,31 +16,40 @@
 
 <body class="page-body">
     <header class="page-header">
-        <a class="main-logo"><img src="img/logo.svg" alt="binabox-logo">
-            <span class="logo-text">BINABOX</span></a>
         <nav class="top-nav">
+            <a class="main-logo"><img src="Images\logo\binaboxlogo36x40.svg" alt="binabox logo" width="188px" height="42px"></a>
             <ul class="nav-list">
                 <li class="nav-item"><a href="index.html">HOME</a>
+                    <button class="dropbtn"><img src="Images\icons-arrows\arrow-down.svg" width="10px" height="5px" alt="arrow down">
+                    </button>
                 </li>
                 <li class="nav-item"><a href="about.html">ABOUT</a>
+                    <button class="dropbtn"><img src="Images\icons-arrows\arrow-down.svg" width="10px" height="5px" alt="arrow down">
+                    </button>
                 </li>
                 <li class="nav-item"><a href="roadmap.html">ROADMAP</a>
+                    <button class="dropbtn"><img src="Images\icons-arrows\arrow-down.svg" width="10px" height="5px" alt="arrow down">
+                    </button>
                 </li>
                 <li class="nav-item"><a href="collection.html">COLLECTION</a>
                 </li>
                 <li class="nav-item"><a href="faqs.html">FAQS</a>
+                    <button class="dropbtn"><img src="Images\icons-arrows\arrow-down.svg" width="10px" height="5px" alt="arrow down">
+                    </button>
                 </li>
                 <li class="nav-item"><a href="pages.html">PAGES</a>
+                    <button class="dropbtn"><img src="Images\icons-arrows\arrow-down.svg" width="10px" height="5px" alt="arrow down">
+                    </button>
             </ul>
             <div class="nav-buttons">
                 <button class="toggle-button" onclick="toggle()">
-                    <img id="sun-icon" src="light-mode-icon.png" alt="Toggle Light/Dark Mode">
+                    <img id="sun-icon" src="Images\buttons\switch.svg" width="24px" height="24px" alt="toggle light or dark mode">
                 </button>
                 <button type="button" class="discord-button">
-                    <img src="#" alt="discord-logo"> DISCORD
+                    <img src="Images\icons\discord.svg" alt="discord icon" width="24px" height="24px"> DISCORD

```

Comment:
Is it really important for the user to know that this is a Discord icon?

Content vs. Decorative Images:

Content Images: These are images that convey important information to the user. For example, a product image, a photo in a news article, or an illustration that is part of the content itself. These images should always have an appropriate alt attribute describing their content.

Decorative Images: These images do not add informational value and are used purely for aesthetic purposes. For example, background patterns, decorative icons, or separators. These should either have an empty alt attribute (alt="") or, better yet, be handled entirely in CSS. If the image is decorative and part of the design, consider using CSS background images or pseudo-elements.